main.js

```diff
@@ -0,0 +1,141 @@
+/* eslint no-param-reassign: ["error", { "props": false }] */
+const boxes = document.querySelectorAll('.box');
+
+const Player = (name, symbol) => ({ name, symbol });
+const player1 = Player('Player 1', 'X');
+const player2 = Player('Player 2', 'O');
+
+const gameBoard = (() => {
+  let board = new Array(9).fill(0);
+
+  const disable = () => {
+    boxes.forEach((box) => {
+      box.disabled = true;
+    });
+  };
+
+  const restart = () => {
```

Comment:
I'm thinking if we can relieve our `board` from having to reset itself? I mean sure we need a fresh board every time we click the Restart button. My idea is to just convert `board` from a module into a factory. We then recreate a new board object, along with new player objects every time we click on Restart.

Since technically, this is a OOP project, we want our `player` and `board` factories to be reusable independent of how we reset the whole application. This way we can extend it easier in the future.